draft-ietf-tls-subcerts.md

```diff
@@ -342,7 +344,9 @@ has the following structure:
 valid_time:
 
 : Time in seconds relative to the beginning of the delegation certificate's
-  notBefore value after which the delegated credential is no longer valid. This MUST NOT exceed 7 days.
+  notBefore value after which the delegated credential is no longer valid.
+  Credentials with valid_times exceeding 7 days will be rejected (as described
+  in {{client-and-server-behavior}}).
```

Comment:
```suggestion
  Endpoints will reject delegate credentials with valid_times exceeding 7 days (as described
  in {{client-and-server-behavior}}).
```

Reply:
Also, the the client-and-server-behavior anchor defined? I checked the source and don't see `{#client-and-server-behavior}` anywhere.

Reply:
It works automatically. Surprise to me too.

Reply:
Hah, cool!